### PR TITLE
Added tests for BackEndDevelopment

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndServicesActions.test.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+		[arrows.nth(8), UrlProvider.urlBuilder(UrlPath.ConsultingServ)],
+	]);
+
+	for (const [arrow, url] of arrowUrlMap) {
+		await baseDriverSteps.checkRedirectToPage(arrow, url, UrlProvider.urlBuilder(UrlPath.BackEndDevelopment));
+	}
+});
+
+test('Check sections expanding and collapsing in "FAQ" from the "Back-End Development" page @Regression @BackEndDevelopment @TSWEB-1208', async () => {
+	const faqContainer = driver.getByTestId(BackEndServices.Faq);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndServicesActions.test.ts:157]: in "FAQ" container

**Review by Valerii-Kryvokhyzha — COMMENTED** [review]
if possible, can you please add a Qase ids to tests?

**Comment by Valerii-Kryvokhyzha** [comment]
if possible, could you please add a Qase ids to tests?

**Comment by KotliarovaK** [comment]
issues in the current brunch

**Pull request closed without merging**